cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1704,17 +1704,22 @@ function start-kube-apiserver {
   if [[ -n "${ENABLE_GARBAGE_COLLECTOR:-}" ]]; then
     params+=" --enable-garbage-collector=${ENABLE_GARBAGE_COLLECTOR}"
   fi
-  if [[ -n "${NUM_NODES:-}" ]]; then
-    # If the cluster is large, increase max-requests-inflight limit in apiserver.
-    if [[ "${NUM_NODES}" -gt 3000 ]]; then
-      params+=" --max-requests-inflight=3000 --max-mutating-requests-inflight=1000"
-    elif [[ "${NUM_NODES}" -gt 500 ]]; then
-      params+=" --max-requests-inflight=1500 --max-mutating-requests-inflight=500"
+  if [[ -n "${NO_THROTTLE_APISERVER:-}" == "true" ]]; then
```

Comment:
Why do we need this change?  I suggest keeping the original code so that we have a comparable configuration with regular Arktos tests